CHANGELOG.asciidoc

```diff
@@ -3,6 +3,38 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.7.1]]
+=== Beats version 7.7.1
+https://github.com/elastic/beats/compare/v7.7.0...v7.7.1[View commits]
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+- Fix `keystore add` hanging under Windows. {issue}18649[18649] {pull}18654[18654]
+
+*Filebeat*
+
+- Unescape file name from SQS message. {pull}18370[18370]
+- Improve cisco asa and ftd pipelines' failure handler to avoid mapping temporary fields. {issue}18391[18391] {pull}18392[18392]
+- Fix source.address not being set for nginx ingress_controller {pull}18511[18511]
```

Comment:
I see the format `foo.bar` in some other entries, not sure if that should be also applied in this case:

```suggestion
- Fix `source.address` not being set for nginx ingress_controller {pull}18511[18511]
```